c++/src/Statistics.hh

```diff
@@ -23,10 +23,17 @@
 #include "orc/OrcFile.hh"
 #include "orc/Reader.hh"
 
+#include "Timezone.hh"
 #include "TypeImpl.hh"
 
 namespace orc {
 
+  struct StatContext {
+    bool correctStats;
+    const Timezone* writerTimezone;
+    StatContext() : correctStats(false), writerTimezone(NULL) {}
```

Comment:
We can change `pointer to const` later too.
 But you are right that it is more efficient to use const for the values too. Will change this.